lib/wallet/test/unit/Cardano/Wallet/DelegationSpec.hs

```diff
@@ -221,10 +227,27 @@ instance Arbitrary EpochNo => Arbitrary WalletDelegationNext where
 instance Arbitrary Withdrawal where
     arbitrary = oneof
         [ WithdrawalSelf <$> arbitrary <*> arbitrary <*> arbitrary
-        , WithdrawalExternal <$> arbitrary <*> arbitrary <*> arbitrary
+        , WithdrawalExternal
+            <$> arbitrary
+            <*> arbitrary
+            <*> arbitrary
+            <*> arbitrary
         , pure NoWithdrawal
         ]
 
+instance Arbitrary XPrv where
+    arbitrary = fromJust . xprvFromBytes . BS.pack <$> vectorOf 96 arbitrary
+
+-- Necessary unsound Show instance for QuickCheck failure reporting
+instance Show XPrv where
+    show = show . xprvToBytes
+
+-- Necessary unsound Eq instance for QuickCheck properties
+instance Eq XPrv where
+    (==) = (==) `on` xprvToBytes
```

Comment:
Why is this instance unsound? Does the `xprvToBytes` function lose information?

Reply:
I just copied it 🤔 not sure... I think it might not be _that_ unsound... I'll remove the comments...